samples/taglist.cpp

```diff
@@ -120,20 +126,22 @@ int main(int argc, char* argv[])
 
             default:
                 rc = 1;
-                break;
+            break;
         }
 
-        if (rc) {
-            std::cout << "Usage: " << argv[0]
-                      << " [--group "
-                         "name|Groups|Exif|Canon|CanonCs|CanonSi|CanonCf|Fujifilm|Minolta|Nikon1|Nikon2|Nikon3|Olympus|"
-                         "Panasonic|Pentax|Sigma|Sony|Iptc"
-                      << "|dc|xmp|xmpRights|xmpMM|xmpBJ|xmpTPg|xmpDM|pdf|photoshop|crs|tiff|exif|aux|iptc]\n"
-                      << "Print Exif tags, MakerNote tags, or Iptc datasets\n";
+        if (rc || bHelp) {
+            std::cout << "Usage: taglist [--help]"                     << std::endl
+                << "           [--group name|"                         << std::endl
+                << "            Groups|Exif|Canon|CanonCs|CanonSi|CanonCf|Fujifilm|Minolta|Nikon1|Nikon2|Nikon3|Olympus|" << std::endl
+                << "            Panasonic|Pentax|Sigma|Sony|Iptc|"                                                        << std::endl
+                << "            dc|xmp|xmpRights|xmpMM|xmpBJ|xmpTPg|xmpDM|pdf|photoshop|crs|tiff|exif|aux|iptc|all|ALL"  << std::endl
+                << "           ]"                                      << std::endl
+                << "Print Exif tags, MakerNote tags, or Iptc datasets" << std::endl
+                ;
         }
-        return rc;
     } catch (AnyError& e) {
         std::cout << "Caught Exiv2 exception '" << e << "'\n";
-        return 1;
```

Comment:
Could we replace the `rc` values with the standard `EXIT_SUCCESS` and `EXIT_FAILURE` values ?